### ISSUE TITLE
Unexpected behaviour of the parameter `colors` of `plot_convex_hull()` from `PolarDiagramMultiSails`

### ISSUE THREAD

**Issue by Loitador41** [hrosailing/hrosailing#52]
**Describe the bug**
When not giving the function any colors at all it just paints all the slices blue. When trying to give it colors, it throws the following error:
`ValueError: dictionary update sequence element #0 has length 5; 2 is required`.

**To Reproduce / Minimal Example**
```python
pd_csv = from_csv("multisails_hro_format_example.csv")
pd_csv.plot_convex_hull((14, 20), colors=('green', 'blue'), show_legend=True)
```

**Comment by Loitador41** [comment]
Also, when trying to pass `(ws, color)` pairs, the graph is being plotted but the colors are completely ignored.

**To Reproduce / Minimal Example**
```python
import matplotlib.pyplot as plt
from hrosailing.polardiagram import PolarDiagramMultiSails as mts
from hrosailing.polardiagram import PolarDiagramTable as tbl

pd_tbl1 = tbl([42, 44, 46], [21, 42, 84], [[1, 2, 3], [4, 5, 6], [7, 8, 9]])
pd_tbl2 = tbl([42, 44, 46], [21, 42, 84], [[10, 11, 12], [13, 14, 15], [16, 17, 18]])
pd_mts = mts((pd_tbl1, pd_tbl2), ("Sail 1", "Sail 2"))

pd_mts.plot_convex_hull(colors=((42, "purple"), (44, "blue"), (46, "red")))
```

**Comment by RobertSchueler** [comment]
Seems to be solved in version 10.0.1 using the newer syntax

```
import matplotlib.pyplot as plt
from hrosailing.polardiagram import PolarDiagramMultiSails as mts
from hrosailing.polardiagram import PolarDiagramTable as tbl

from hrosailing.plotting import plot_polar

pd_tbl1 = tbl([42, 44, 46], [21, 42, 84], [[1, 2, 3], [4, 5, 6], [7, 8, 9]])
pd_tbl2 = tbl([42, 44, 46], [21, 42, 84], [[10, 11, 12], [13, 14, 15], [16, 17, 18]])
pd_mts = mts((pd_tbl1, pd_tbl2), ("Sail 1", "Sail 2"))

plot_polar(pd_mts, ws=[42, 44, 46], colors=((42, "purple"), (44, "blue"), (46, "red")), use_convex_hull=True)

plt.show()
```
